src/common-events/utils/headless-cms/headlessCmsUtils.tsx

```diff
@@ -141,6 +141,8 @@ export const getDefaultCollections = (
         description: collection.description,
         type: getCollectionUIType(collection),
         collectionContainerProps: { withDots: true },
+        // TODO: The collection specific URL from the module itself should be used instead of a hard coded value.
+        showAllUrl: '/',
```

Comment:
so do we need the actual value right now, or we use empty instead and hiding the link for right now?

Reply:
when the url is not given, no link is shown. The link should come from the CMS inside the module attributes. It's ordered, but not implemented properly yet -- as far as I know